bokehjs/prepare.js

```diff
@@ -12,7 +12,3 @@ function write_hash(file) {
 write_hash("package.json")
 
 const chalk = require("chalk")
```

Comment:
This import should be removed as well.

Reply:
done in https://github.com/bokeh/bokeh/pull/8545/commits/d037b17ab39d22b79b7b8c00e9080f20f0010891